src/middlewared/middlewared/plugins/service_monitor.py

```diff
@@ -273,18 +273,21 @@ def __init__(self, *args):
         for s in services:
             thread_name = s['sm_name']
             s_config = None
+            service_enabled = False
             # Remove stale alerts
             ServiceMonitorThread.reset_alerts(thread_name)
 
-            if not s['sm_enable']:
-                self.logger.debug("[ServiceMonitorService] skipping %s", thread_name)
-                continue
-
             if thread_name in ('activedirectory', 'ldap', 'nis'):
-                s_config = await self.middleware.call('datastore.query', f'directoryservice.{thread_name}',
-                                                      None, {'get': True})
+                service_name = 'ad' if thread_name == 'activedirectory' else thread_name
+                s_config = await self.middleware.call('datastore.query', f'directoryservice.{thread_name}', None, {'get': True})
+                service_enabled = s_config[f"{service_name}_enable"]
             else:
-                s_config = await self.middleware.call(f'{thread_name}.config')
+                s_config = await self.middleware.call('service.query', [('service', '=', f"{thread_name}")], {'get': True})
+                service_enabled = s_config['enable']
```

Comment:
`enable` here only means "start on boot" checkbox enabled.

We could have the service running but that checkbox disabled. We still dont have a way to tell if a service has been started by a user but its not currently running.

Is that expected?

Also which services do we monitor that are not directory services?